parsec/core/gui/instance_widget.py

```diff
@@ -47,6 +49,26 @@ async def _do_run_core(config, device, qt_on_ready):
                 await trio.sleep_forever()
 
 
+def check_macfuse_version() -> bool:
+    import os
+
+    macfuse_plist_path = "/Library/Filesystems/macfuse.fs/Contents/Info.plist"
+    if not os.path.exists(macfuse_plist_path):
+        # If macFUSE is installed, this file should exist at this path. If not,
+        # it probably means that its previous version, osxfuse, is installed
+        # instead (in a "osxfuse.fs" directory). Any current MacOS version is
+        # incompatible with osxfuse.
+        return False
+
+    import plistlib
+
+    local_version = plistlib.readPlist(macfuse_plist_path)["CFBundleVersion"]
+
+    if local_version < min_macfuse_version:
```

Comment:
comparing strings is dangerous !

`1.10 > 1.2` but `"1.10" < "1.2"`

We should use `from packaging.version.Version` (see what is done in `parsec/core/gui/new_version.py`)